submit-api/src/submit_api/services/package.py

```diff
@@ -342,6 +342,23 @@ def _resubmit_package(cls, package, session):
         cls._log_activity_submission(package, ActivityActionType.UPDATED_SUBMISSION.value, session)
         return package
 
+    @classmethod
+    def accept_update_request(cls, package_id):
+        """Submit the package by updating its status and items."""
+        with session_scope() as session:
+            package = cls._get_and_validate_complete_package(package_id)
+
+            current_app.logger.info(f"Accepting update reuqest for package {package.id}")
+            open_update_requests = [
+                request for request in package.update_requests
+                if request.status in {UpdateRequestStatus.OPEN.value,
```

Comment:
Do we wanna accept update requests that are not even pending?

When EAO create an update request the status is OPEN
When Entity resubmits package the status becomes PENDING

I think it makes sense to only accept pending requests

Reply:
Updated